### PR TITLE
Laurin/fix delete

### PR DESCRIPTION
**Pull request by DeNic0la**

### CODE REVIEW

File: movie/controller.js [hunk 1]
```diff
@@ -11,7 +11,10 @@ export async function listAction(request, response) {
 export async function removeAction(request, response) {
     const id = parseInt(request.params.id, 10);
     await remove(id,request.user.id);
-    response.redirect(request.baseUrl);
+
+    const data = await getAllMoviesWithRating(request.user.id);
+    const body = render(data);
+    response.send(body)
```
Comment (DeNic0la) [inline movie/controller.js]: this wasn't really necessary

File: movie/model.js [hunk 2]
```diff
@@ -35,7 +35,9 @@ export async function get(id, uid) {
 
 export async function remove(id, uid) {
     let m = await Movies.findByPk(id);
+    let r = await Ratings.findAll({ where: { movie: id } });
     if (m.public === true || parseInt(m.user) === parseInt(uid)) {
+        await r.destroy();
```
Comment (DeNic0la) [inline movie/model.js]: this looks like it should work

**Review by DeNic0la — COMMENTED** [review]
I Tested it and it doesn't work

**Comment by DeNic0la** [comment]
This Closes #14 and fixes other things.
I Tested it and delete is working.

**Review by fabianmueller7 — APPROVED** [review]
tested and works!

**Pull request merged**